### ISSUE TITLE
Empty 'comments_full' in live post 

### ISSUE THREAD

**Issue by hungphamvn** [kevinzg/facebook-scraper#575]
Hi @neon-ninja,
I'm trying to get comments from live posts:
```
next(get_posts(
            post_urls=["100066517395693"],
            cookies=os.path.join(os.getcwd(), "../cookies.txt"),
            options={'comments': True}
        ))
```
but can't get any comment. 
It's working fine with normal posts.
Could you please take a look? 
Thanks a ton!

**Comment by neon-ninja** [comment]
post_urls expects single posts. This ID points to a profile. Don't use post_urls in that case. Like so: ```python
post = next(get_posts("100066517395693", options={'comments': True}))
```

**Comment by hungphamvn** [comment]
You saved my day, Nick.
One more thing is I'm trying to get  all comments from live stream posts which has a lot of comments (>10k, such as this one: 245717210878706), but it only return ~5k comments.
Could you please help me out?

**Comment by neon-ninja** [comment]
At this sort of scale, you should probably set `options={"comments": "generator"}`, keep track of the comment pagination URL, handle the TemporarilyBanned exception, etc. See https://github.com/kevinzg/facebook-scraper/issues/409#issuecomment-890613544

**Comment by hungphamvn** [comment]
Yeah, I've just tried it. Seems facebook has an issue with view more comments (or maybe their feature), so we cannot get fully amount of post's comments :(

**Comment by neon-ninja** [comment]
I don't think it's feasible for a human to sit down and read 5K comments so probably not a big problem for them

**Comment by hungphamvn** [comment]
> I don't think it's feasible for a human to sit down and read 5K comments so probably not a big problem for them

yeah, you're absolutely right. but are there any work around solution for this case?

**Comment by neon-ninja** [comment]
Not to my knowledge. I wonder if the desktop version suffers from the same issue or not.

**Comment by hungphamvn** [comment]
@neon-ninja, Yeah the desktop version has the same issue.
And about this one:
https://github.com/kevinzg/facebook-scraper/issues/575#issuecomment-977207198
It's working with ID points to a profile, but what should I do  if it point to page account?

**Comment by neon-ninja** [comment]
Just pass in the URL fragment or page ID. Like in the readme.

**Comment by hungphamvn** [comment]
> Just pass in the URL fragment or page ID. Like in the readme.

It seems not working with live stream post with page account?

**Comment by neon-ninja** [comment]
Which page are you having this problem with?

**Comment by hungphamvn** [comment]
> 

yeah. I'm trying to get full comments from live streaming from page (e.g: https://www.facebook.com/tongkhobuonluyennguyen/videos/267475575351087/)
and when I try like this:
```
post = next(
                get_posts(
                    post_urls=[267475575351087],
                    cookies=os.path.join(os.getcwd(), "cookies.txt"),
                    options={
                        "comments": "generator"
                    },
                )
            )
```
I can only get a few comments but not all of them (it's working if live stream points to a personal account)
Any idea?

**Comment by neon-ninja** [comment]
I see - try this - https://github.com/kevinzg/facebook-scraper/commit/93d890cc6042ede46ed5a511e6673d9472d5dedc

**Comment by hungphamvn** [comment]
> I see - try this - [93d890c](https://github.com/kevinzg/facebook-scraper/commit/93d890cc6042ede46ed5a511e6673d9472d5dedc)
Yeah, This seems a lot better but there is another issue with Facebook on the mobile version that they didn't display the newest comments (just weird way)
